src/pages/PrivacyPolicyPage/index.tsx

```diff
@@ -0,0 +1,202 @@
+import { useEffect, type FC } from 'react';
+import { useNavigate } from 'react-router';
+
+export const PrivacyPolicyPage: FC = () => {
+  const navigate = useNavigate();
+
+  const handleClick = (): void => {
```

Comment:
```suggestion
  const onGoBackClick= (): void => {
      navigate('/');
  }
```
or add condition, because if we had new window, we would go to "about:blank" page